<launch>
    <node pkg="seekurjr_navigation" name="simple_navigation_goals"  type="simple_navigation_goals" output="screen">
        <rosparam file="$(find seekurjr_navigation)/params/simple_navigation_goals.yaml" command="load" />
    </node>
</launch>
